Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -224,24 +225,33 @@ def getKerningPairs(self, side1Classes, side2Classes):
 
         return result
 
-    def _determineFontScripts(self):
+    def _determineFontScripts(self, feaScripts):
```

Comment:
Can you move this to the base class or other common place? I’d like to use this in the mark writer as well, so both writers have consistent behavior. May be include the call